metric/system/process/process_test.go

```diff
@@ -357,6 +360,10 @@ func TestProcCpuPercentage(t *testing.T) {
 }
 
 func TestIncludeTopProcesses(t *testing.T) {
+	// noop line so `runThreads` is not marked as unsued for CI on linux
+	// can't use linter directives, since they'll fail on platforms where `runthreads` IS used
```

Comment:
You should be able to turn off the lint warning for unused directives.

Reply:
https://github.com/elastic/elastic-agent-system-metrics/blob/085e4529f3c4f91dd377cadbbe7a2bf321989438/.golangci.yml#L102-L104

Reply:
ah! found the config option, yah